config-linux.md

```diff
@@ -196,6 +196,39 @@ For example, to run a new process in an existing container without updating limi
 
 Runtimes MAY attach the container process to additional cgroup controllers beyond those necessary to fulfill the `resources` settings.
 
+### Cgroup ownership
+
+Runtimes MAY change (or cause to be changed) the owner of the
+container's cgroup to the host uid that maps to uid 0 in the
+container's user namespace, according to the following rules.
+
+Runtimes MUST NOT change the ownership of container cgroups when
+cgroups v1 is in use.  Cgroup delegation is not secure in cgroups
+v1.
+
+Runtimes SHOULD change the cgroup ownership if and only if the
+cgroup filesystem is to mounted read/write; that is, when the
+configuration's `mounts` array contains an object where:
+
+- The `source` field is equal to `"cgroup"`
+- The `destination` field is equal to `"/sys/fs/cgroup"`
+- The `options` field does not contain the value `"ro"`
+
+If the configuration does not specify such a mount, the runtime MUST
+NOT change the cgroup ownership.
```

Comment:
Same comment about MUST NOT here -- but in addition, the use of the phrase "if and only if" above makes this MUST NOT requirement a contradiction -- if I SHOULD do something if and only if X, then if not X I SHOULD NOT do it.

Reply:
relaxed to SHOULD.